brian-chelsea/public/scripts/controllers/aboutController.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+var app = app || {};
+
+(function(module) {
+  const aboutController = {};
+
+  // done: Define a function that hides all main section elements, and then reveals just the #about section:
+  function about(){
+    $('#articles').hide();
+    $('about').show();
+  }
```

Comment:
This function needs to be attached to the controller in order to accessible outside the IIFE